projects/github.com/helmfile/vals/package.yml

```diff
@@ -0,0 +1,31 @@
+distributable:
+  url: https://github.com/helmfile/vals/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+display-name: pinniped
+
+versions:
+  github: helmfile/vals/releases/tags
+
+provides:
+  - bin/vals
+
+build:
+  dependencies:
+    go.dev: '*'
+  script:
+   - go mod download
+   - mkdir -p "{{ prefix }}"/bin
+   - go build -trimpath -ldflags="$LDFLAGS" -o "{{ prefix }}"/bin/vals ./cmd/vals
+  env:
+    CGO_ENABLED: 0
+    LDFLAGS:
+      - -w
+      - -s
+      - -X main.version=v{{version}}
+
+test:
+  - vals --help | grep "vals is a Helm-like configuration"
+  # Some versions of vals had no command to print the version
+  - echo $(vals version || true) | grep "v{{version}}" || \
```

Comment:
likely need a `vals version 2>&1 || true...` here.

Reply:
It looks like `vals --help` outputs to stderr. Updated